src/rules/noAnyRule.ts

```diff
@@ -38,9 +40,14 @@ export class Rule extends Lint.Rules.AbstractRule {
 
             Also see the \`no-unsafe-any\` rule.
         `,
-        optionsDescription: "Not configurable.",
-        options: null,
-        optionExamples: [true],
+        optionsDescription: Lint.Utils.dedent`
+            If \`"${OPTION_IGNORE_REST_ARGS}"\` is provided rest arguments will be ignored.
+        `,
+        options: {
+            type: "string",
```

Comment:
TSLint's been moving towards having options as objects with members, rather than single primitives. They're a lot easier to add more options to later on and a bit more clear in `tslint.json`s. You'll want to switch this to `type: "object"`; see `memberOrderingRule.ts` and `noShawdowedVariableRule.ts` as examples.